### ISSUE TITLE
Many Accessibility Issues

### ISSUE THREAD

**Issue by maddaxlallatin** [datacamp/datacamp-light#230]
Data camp use in a website creates many accessibility issues such as low contrast, empty button, missing form labels, missing alt text among many others.
